test/request_cache/metrics_test.exs

```diff
@@ -5,43 +5,91 @@ defmodule RequestCache.TelemetryMetricsTest do
 
   alias RequestCache.Support.Utils
 
+  @expected_ttl 3600000
   @expected_measurements %{count: 1}
   @expected_rest_cache_hit_event_name [:request_cache_plug, :rest, :cache_hit]
   @expected_rest_cache_miss_event_name [:request_cache_plug, :rest, :cache_miss]
   @expected_graphql_cache_hit_event_name [:request_cache_plug, :graphql, :cache_hit]
   @expected_graphql_cache_miss_event_name [:request_cache_plug, :graphql, :cache_miss]
+  @expected_graphql_cache_put_event_name [:request_cache_plug, :graphql, :cache_put]
   @expected_cache_put_event_name [:request_cache_plug, :cache_put]
 
+  @miss_cache_key "/graphql:BE1120D4C931B50910C1B8788FA21108"
+  @hit_cache_key "/graphql:14BFE314D845C31342E288408A7DACE4"
+
+  @expected_cache_miss_metadata %{ttl: @expected_ttl, cache_key: @miss_cache_key, labels: [:graphql, :test_endpoint]}
+  @expected_cache_hit_metadata %{ttl: @expected_ttl, cache_key: @hit_cache_key, labels: [:graphql, :test_endpoint]}
+
   setup do: %{self: self()}
 
   describe "GraphQL RequestCache.Plug.call/2" do
-    test "cache miss", %{self: self, test: test} do
+    setup do
+      conn = Utils.graphql_conn()
```

Comment:
Don't pipe this! One pipe means no pipe

Reply:
dangit, missed that one in the cleanup pass, thanks for catching!